swagger/v1/get_journey.yaml

```diff
@@ -32,11 +32,12 @@ GetJourney:
         state:
           type: string
           enum:
+          - proposed
```

Comment:
are we missing the `rejected` state here?

Reply:
oops - my bad 🤦